### ISSUE TITLE
SNMP get: 

### ISSUE THREAD

**Issue by Sergio-tix** [node-red/node-red-nodes#969]
### Which node are you reporting an issue on?
- node-red-node-snmp v1.0.5 (SNMP Get)

### What are the steps to reproduce?
- Reading SMART SSD, parameter № 241 "Total_LBAs_Written"

### What happens?
- I get an error: "RangeError: Read integer 3579457891 is outside the signed 32-bit range"

### What do you expect to happen?
- I want to get the real value which is "12150544793". In the future, this value may increase by one more digit.
- 
### Please tell us about your environment:

- [ ] Node-RED version: 3.0.2
- [ ] node.js version: 18.12.1
- [ ] npm version: 9.0.2
- [ ] Platform/OS: Synology (DVA1622 DSM v.7.1.1 ) Docker - Ubuntu 20.04.5 LTS
- [ ] Browser: Google Chrome 108.0.5359.125, x64

Need to change the type of the variable to get huge numbers.

**Comment by dceejay** [comment]
can you attach a copy of the node-red log file - that should also show the error with a bit more context. Certainly a signed 32 bit Int can't be greater than about 2.1 million so it is correct in what it is reporting ...  but it's not clear if it's the snmp library or something we are doing with it afterwards.

**Comment by Sergio-tix** [comment]
The node-red LOG has exactly the same information and nothing more:
![SNMP Get 2](https://user-images.githubusercontent.com/18720428/211788906-5d3faef7-782e-4304-819e-18ab35f69303.PNG)

![SNMP Get](https://user-images.githubusercontent.com/18720428/211788974-26363caf-7439-4683-905d-869c800c7726.PNG)

```
[{"id":"31d93ddc.b29162","type":"snmp","z":"975b3811b4e20b9b","host":"192.168.15.5","version":"1","timeout":5,"community":"public","auth":"noAuthNoPriv","authprot":"MD5","privprot":"DES","oids":"1.3.6.1.4.1.6574.5.1.1.8.14\n","name":"","credentials":{},"x":1290,"y":240,"wires":[["279326ae.a27b6a"]]},{"id":"40af2498.6eaf7c","type":"inject","z":"975b3811b4e20b9b","name":"","props":[{"p":"payload"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"true","payloadType":"bool","x":1130,"y":240,"wires":[["31d93ddc.b29162"]]},{"id":"279326ae.a27b6a","type":"debug","z":"975b3811b4e20b9b","name":"","active":true,"tosidebar":true,"console":false,"tostatus":true,"complete":"payload","targetType":"msg","statusVal":"payload","statusType":"auto","x":1470,"y":240,"wires":[]}]
```
ssd SMART
![ssd](https://user-images.githubusercontent.com/18720428/211789691-8fff9c21-d1a7-449b-89b8-011d45d73711.png)

**Comment by dceejay** [comment]
Yes I think it may well be an error in the MIB definition of your device eg https://www.reddit.com/r/synology/comments/q2x7bv/synology_smart_mib_oid_13614165745_uses_only/
So not much we can do about it.

**Comment by Sergio-tix** [comment]
![erf](https://user-images.githubusercontent.com/18720428/211808821-037bd075-a9fa-4779-9a0d-75986d7a3c85.png)
Yes. Apparently this is a Synology bug. I am 100% sure of this, I have been using their products for a long time. This bug will not be fixed soon. But in any case, even if Synology fixes the problem, the node will still not accept this value.
We need to fix the node so that it is ready to accept large numbers. INTEGER64 would be enough for a long time. I went to write in support of Synology.

**Comment by iprime2** [comment]
Hey, is this issue resolved @Sergio-tix  @dceejay 
Can you tell how can I fix this isue.

**Comment by dceejay** [comment]
Hi. Not unless Synology have fixed it, as they have defined it incorrectly in their MIB

This assumes you have the same hardware etc
